src/docs/_components/Placeholder/Placeholder.tsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import { classNames } from '../../../lib/utils/classNames';
+import styles from './Placeholder.scss';
+import {
+  PlaceholderProps, CustomCSSProperties,
+} from './Placeholder.types';
```

Comment:
I think this should either be a single line, or each item should have its own line. Why ESlint doesn't cover it…? @bedrich-schindler @mbohal